tests/selenium/single-page-stripe-test.js

```diff
@@ -11,6 +11,11 @@ module.exports = function(driver, By, done) {
     });
   });
   driver.findElement(By.id('email')).sendKeys('send-donation@test.com');
+  driver.findElement(By.id('billing-name')).sendKeys('a');
```

Comment:
Just updating the test, which now uses a bunch of new fields so we need to fill them in.